main.cpp

```diff
@@ -1,10 +1,27 @@
+#include "main.h"
+
 #include <iostream>
+#include <string>
 
 #include <GL/glew.h>
 #include <GLFW/glfw3.h>
 #include <glm/glm.hpp>
 #include <glm/gtc/matrix_transform.hpp>
 
+#include "indexbuffer.h"
```

Comment:
Swap the ordering of your includes. The general ordering is recommended to be:

#include "headerFileOfThisSourceFile.h" // doesn't apply here, but in classes etc

#include "myprojheader1.h"
#include "myprojheaderN.h"

#include <TPSHeader1.h> // GL/glew etc
...

#include <stlHeader.h> // string, etc

https://softwareengineering.stackexchange.com/a/325565